iohk/NixOps.hs

```diff
@@ -828,7 +834,7 @@ startForeground o c node =
     ssh o c "bash" ["-c", Arg $ "'sudo -u cardano-node " <> unitStartCmd <> "'"] node
 
 stop :: Options -> NixopsConfig -> IO ()
-stop o c = echo "Stopping nodes..."
+stop o c = echo (unsafeTextToLine $ "Stopping " <> (fst $ scopeNameDesc o c))
```

Comment:
Initially I was a bit concerned by this `unsafeTextToLine` function, but it simply means Turtle doesn't bother verifying the argument has no newlines. In this context the risk of introducing newlines is low.